Sources/Fullscreen/AddressView/AddressView.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+public protocol AddressViewDelegate: class {
+    func addressViewDidSelectCopyButton(_ addressView: AddressView)
+    func addressViewDidSelectGetDirectionsButton(_ addressView: AddressView)
+    func addressViewDidSelectCenterMapButton(_ addressView: AddressView)
+    func addressView(_ addressView: AddressView, didSelectMapTypeAtIndex index: Int)
+}
+
+@objc public class AddressView: UIView {
```

Comment:
Do we need to expose this class to objc?

Reply:
We don't, fixing :)